tests/test_sailor/test_pai/__init__.py

```diff
@@ -3,7 +3,7 @@
 import importlib
 
 # make sure all pai modules are imported before the tests are run
-# otherwise the utils tests for ResultSet relying on __subclasses__ will not include all subclasses,
+# otherwise the utils tests for AssetcentralEntityCollection relying on __subclasses__ will not include all subclasses,
```

Comment:
I'm somewhat confident that you can remove the contents of this file now. Same for `test_assetcentral/__init__.py`.

Reply:
I've taken another look around and also believe so :)